applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_analysis.py

```diff
@@ -2,15 +2,19 @@
 
```

Comment:
I am thinking that in order to improve the organization all the `adaptive_remeshing` things can go into a `adaptive_remeshing` folder
same I wanted to do for a while now with the Solvers, but haven't gotten to do it yet

Reply:
OK

Reply:
Done